packages/flutter_nekoton_bridge/lib/nekoton/core/accounts_storage/models/account_to_add.g.dart

```diff
@@ -9,17 +9,31 @@ part of 'account_to_add.dart';
 _$_AccountToAdd _$$_AccountToAddFromJson(Map<String, dynamic> json) =>
     _$_AccountToAdd(
       name: json['name'] as String,
-      publicKey: json['publicKey'] as String,
+      publicKey: publicKeyJsonConverter.fromJson(json['publicKey'] as String),
       contract: WalletType.fromJson(json['contract'] as Map<String, dynamic>),
       workchain: json['workchain'] as int,
-      explicitAddress: json['explicitAddress'] as String?,
+      explicitAddress: _$JsonConverterFromJson<String, Address>(
```

Comment:
Im talking about this. Before, it tried to take `explicitAddress` from object, but now it will try to take `address` as in `addressJsonConverter`